scripts/asm-installer/tests/fake_kubectl

```diff
@@ -2,6 +2,35 @@
 
 FAKE_CONFIG="$(cat "${KUBECONFIG}")"
 
+if [[ "${*}" == *"version"* ]]; then
+  cat <<EOF
+{
+  "clientVersion": {
+    "major": "1",
+    "minor": "19",
+    "gitVersion": "v1.19.2",
+    "gitCommit": "f5743093fd1c663cb0cbc89748f730662345d44d",
+    "gitTreeState": "clean",
+    "buildDate": "2020-09-16T13:41:02Z",
+    "goVersion": "go1.15",
+    "compiler": "gc",
+    "platform": "linux/amd64"
+  },
+  "serverVersion": {
+    "major": "1",
+    "minor": "16+",
```

Comment:
Is this test just for k8s 1.16?

Reply:
Yea we'll probably have to update it next version. If you think there's a better way then lmk and we can open an issue or something.